packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -8,6 +8,7 @@ import 'dart:math' as math;
 import 'package:flutter/foundation.dart';
 import 'package:flutter/rendering.dart';
 
+import '../rendering/sliver_group.dart';
```

Comment:
This should not be necessary since you are already importing `rendering.dart`. 
Looks like `sliver_group.dart` needs to be added as an export in `rendering.dart`

Reply:
Fixed!